src/Libraries/DSCPython/DSCPythonCodeCompletionDataCore.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Autodesk.DesignScript.Interfaces;
+
+namespace DSCPython
+{
+    /// <summary>
+    /// Concrete type that gets returned and converted to an Avalonedit type implementing
+    /// ICompletionData when used from WPF ScriptEditorContorl.
+    /// </summary>
+    internal class DSCPythonCodeCompletionDataCore : IExternalCodeCompletionData
+    {
+        private readonly IExternalCodeCompletionProviderCore provider;
+        private string description;
+
+        public DSCPythonCodeCompletionDataCore(string text, string stub, bool isInstance,
+                                                ExternalCodeCompletionType completionType, IExternalCodeCompletionProviderCore providerCore)
+        {
+            Text = text;
+            Stub = stub;
+            IsInstance = isInstance;
+            provider = providerCore;
+            CompletionType = completionType;
+        }
+
+        public string Text { get; private set; }
```

Comment:
@reddyashish Do you make these properties public because they are made so in IronPythonCodeCompletionData before? Can we make them internal for now, and maybe add some comments for each of them?